### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by yayaismaail**
In this milestone I:

- Used primary key & foreign key mechanism for joining tables.
- Queried multiple tables.
- Prepared complex queries that answer analytical questions.
- 
vet1
vet2

**Above are the screenshot required by previous reviewer**

vet1
vet2
vet3
vet4
vet5
vet6
vet7
vet8
vet9
vet10
vet11

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+
+CREATE TABLE species (
+    id integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+ALTER TABLE animals
+DROP COLUMN species,
+ADD COLUMN species_id INT,
+ADD COLUMN owner_id INT;
```
Comment (Munsa1) [inline schema.sql]: - [ ] Kindly add the referential integrity constraint for the 2 columns you have added to the animals table


e.g
```
ALTER TABLE animals ADD species_id int REFERENCES species(id);
ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,116 @@
...
+SELECT o.full_name, COALESCE(GROUP_CONCAT(a.name), 'None') AS owned_animals
+FROM owners AS o
+LEFT JOIN animals AS a ON o.id = a.owner_id
+GROUP BY o.full_name;
+
+SELECT s.name, COUNT(*) AS animal_count
+FROM animals AS a
+JOIN species AS s ON a.species_id = s.id
+GROUP BY s.name;
+
```
Comment (Munsa1) [inline queries.sql:99]: - [ ] Kindly add a screenshot to the PR description as per the requirement for the query `List all owners and their animals, remember to include those that don't own any animal.`

e.g
image

File: README.md [hunk 3]
```diff
@@ -1,236 +1,97 @@
...
+👤 **Ismail Halliru Muhammad**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/yayaismaail)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/IsmailhalliruM1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/ismail-halliru-muhammad-2a8453127)
```
Comment (Munsa1) [inline README.md]: - [ ] Kindly fix the link to your `LinkedIn` profile for a more descriptive ReadMe file
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added All Queries :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @yayaismaail , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights


✅ Implemented missing foreign keys in their corresponding tables
✅ Employed informative Git commit messages 👍

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**